internal/turso/turso.go

```diff
@@ -25,6 +25,7 @@ type Client struct {
 	Feedback      *FeedbackClient
 	Organizations *OrganizationsClient
 	Locations     *LocationsClient
+	Tokens        *TokensClient
```

Comment:
Where is TokensClient defined?

Reply:
On `internal/turso/token.go`. But it wasn't there. Rebase hell made it end up in the location PR by accident.